### DISCUSSION TITLE
[BUG] Readme is often hidden

### DISCUSSION THREAD

**Discussion by zdu-strong** [npm/feedback#840] [Website] [answered]
Readme is often hidden.
You can access any npm package, which will have a chance to trigger. For example: https://www.npmjs.com/package/rxjs
Actually it has reademe file.

![readme](https://user-images.githubusercontent.com/48673442/210082544-368876c6-90d5-4316-ac06-07a7d71bdfac.png)
![SometimeReadme](https://user-images.githubusercontent.com/48673442/210082572-2859e465-9166-41b6-96bc-6c856a605d2b.png)

**Comment by aminnairi** [comment]
This happens a lot on when browsing on my mobile device when checking for my NPM packages that have a README without any known cause clearly identified. Sometimes it just does not show the documentation at all without the warrning about the package not having any documentation yet.

**Reply by zdu-strong** [reply]
I add a issue in [npm/documentation](https://github.com/npm/documentation/issues/375)

**Reply by aminnairi** [reply]
Looks like the issue has been closed.

**Reply by zdu-strong** [reply]
It looks like this is the repository of the website, it has been archived.
https://github.com/npm/www

**Reply by aminnairi** [reply]
Yes indeed, I could not find anything that looks like an active maintained repository for the website. I did ask for the link to the said repository though. Waiting for an answer.

**Reply by ljharb** [reply]
The website code has been private for a number of years. This discussion, or filing a support ticket, is the proper way to convey an issue.

**Reply by MikeMcC399** [reply]
I am seeing this issue in https://www.npmjs.com/package/@cypress/github-action?activeTab=readme as well.

It seems to alternate between displaying the readme, and displaying no readme text.

Is there any explanation for this?

**Reply by MikeMcC399** [reply]
@zdu-strong 

Could you check using Google Chrome incognito mode? It looks like it always displays the readme in incognito mode, which would point to a browser cookie issue (or similar).

**Reply by zdu-strong** [reply]
Yes, using incognito mode is showing the readme. But the bug still needs to be solved. What do you think?

**Reply by MikeMcC399** [reply]
@zdu-strong 
> Yes, using incognito mode is showing the readme. But the bug still needs to be solved. What do you think?

It's good to know that you get the same results. This bug should definitely be addressed. Who can fix it?

**Reply by MikeMcC399** [reply]
I guess it is a caching issue. At the moment I can't reproduce it. The next time it happens I will try [Shift + F5 in Google Chrome](https://support.google.com/chrome/answer/157179#zippy=%2Cwebpage-shortcuts) for "Reload the current page, ignoring cached content".

**Reply by MikeMcC399** [reply]
It seems to be a refresh / caching issue. When I click on the Readme tab nothing reloads, however a simple F5 displays the README contents. This however does not always work. Sometimes I need Shift + F5 to get the README to display.

**Comment by MylesBorins** [comment, marked as answer]
Please file a support ticket about this issue to https://npmjs.com/support and our team will ensure this gets properly routed to engineering to look at

**Reply by MikeMcC399** [reply]
@zdu-strong 

Will you open a support ticket as suggested by @MylesBorins ?

**Reply by MikeMcC399** [reply]
@zdu-strong 

Since I didn't hear back from you I went ahead and logged a support ticket. The issue affects me as well, so I am interested in getting a resolution.

@MylesBorins 

Thanks for your advice. I have logged Ticket ID: 1995222

**Reply by MikeMcC399** [reply]
The response of npm support was that this is most likely a caching issue and the recommended solution is to clear browser data.

So in Google Chrome I went to the three-dot menu, top-right > More Tools > Clear Browsing Data, Basic / All time and selected "Clear data".

I will keep an eye out if the problem reappears.

**Reply by zdu-strong** [reply]
> The response of npm support was that this is most likely a caching issue and the recommended solution is to clear browser data.
> 
> So in Google Chrome I went to the three-dot menu, top-right > More Tools > Clear Browsing Data, Basic / All time and selected "Clear data".
> 
> I will keep an eye out if the problem reappears.

Very good, thank you. I cleared the cache, will pay attention to whether this problem will appear in the future

**Reply by zdu-strong** [reply]
Unfortunately, the readme is hidden again

**Reply by MikeMcC399** [reply]
@zdu-strong 

> Unfortunately, the readme is hidden again

I had the same experience today, so I will reply to npm support that they need to look into the issue.

**Reply by MikeMcC399** [reply]
There was possibly an issue with Cloudflare CDN. I can regularly see the Readme now.

**Reply by zdu-strong** [reply]
Good news, the bug is not reproduced.
I think it has been fixed.

**Reply by vishwas-r** [reply]
The bug seems to have re-appeared! It happened to me with CanvasJS angular stockcharts package ([https://www.npmjs.com/package/@canvasjs/angular-stockcharts](https://www.npmjs.com/package/@canvasjs/angular-stockcharts))
![NPM Package - Readme not showing](https://img001.prntscr.com/file/img001/8M2jnywGSkK3cH5DtarCqA.png)

**Reply by Steve0929** [reply]
I'm having the same issue with https://www.npmjs.com/package/tiktok-tts it is intermittently showing the readme.md

**Comment by MikeMcC399** [comment]
@vishwas-r 

There seems to be another error with the npm registry at this time.
https://www.npmjs.com/package/@canvasjs/angular-stockcharts?activeTab=versions then clicking on https://www.npmjs.com/package/@canvasjs/angular-stockcharts/v/1.0.3 was giving "error loading page"

I tried with another package and got the same error.
